src/main/groovy/net/minecraftforge/gradleutils/GradleUtilsExtension.groovy

```diff
@@ -6,25 +6,22 @@ package net.minecraftforge.gradleutils
 
 import groovy.transform.CompileStatic
 import org.gradle.api.Project
-import org.gradle.api.file.Directory
-import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.provider.Provider
 
 import javax.inject.Inject
 
 @CompileStatic
 class GradleUtilsExtension {
     private final Project project
-    final DirectoryProperty gitRoot
-    private final Provider<Map<String, String>> gitInfo
```

Comment:
Removal of gitRoot is a braking change

Reply:
Is it actually? My IDE marked it as private. I can add it back, but it quite literally serves no purpose.